charts/ingress-nginx/templates/check.yaml

```diff
@@ -0,0 +1 @@
+{{ include  "isControllerTagValid" . | required "controller container image tag should be 0.27.0 or higher" }}
```

Comment:
Instead of including this in its own file I think that we should have the include function in the daemonset/deployment.

Reply:
done